xds/src/main/java/io/grpc/xds/LookasideChannelLb.java

```diff
@@ -123,30 +123,28 @@ public void shutdown() {
     lbChannel.shutdown();
   }
 
-  // TODO(zdapeng): The old AdsStreamCallback will be renamed to LookasideChannelCallback,
-  // and AdsStreamCallback2 will be renamed to AdsStreamCallback
   /**
    * Callback on ADS stream events. The callback methods should be called in a proper {@link
    * io.grpc.SynchronizationContext}.
    */
-  interface AdsStreamCallback2 {
+  interface AdsStreamCallback {
```

Comment:
This should be moved to `XdsComms2.java`. Callback interfaces should be defined in caller classes and implemented in classes that provide the implementation to callers.

Reply:
Yes, given that this is going to be rewritten, I didn't move. But I'm OK to move.

Reply:
Done.